src/java/apiview-java-processor/pom.xml

```diff
@@ -50,14 +50,14 @@
     <dependency>
       <groupId>org.mockito</groupId>
       <artifactId>mockito-core</artifactId>
-      <version>4.0.0</version> <!-- {x-version-update;org.mockito:mockito-core;external_dependency} -->
+      <version>4.11.0</version> <!-- {x-version-update;org.mockito:mockito-core;external_dependency} -->
       <scope>test</scope>
     </dependency>
-      <dependency>
-          <groupId>commons-lang</groupId>
-          <artifactId>commons-lang</artifactId>
-          <version>2.6</version>
-      </dependency>
+    <dependency>
+      <groupId>org.unbescape</groupId>
+      <artifactId>unbescape</artifactId>
```

Comment:
Should we use [commons-lang3](https://mvnrepository.com/artifact/org.apache.commons/commons-lang3/3.14.0) which seems to be well-maintained compared to [unbescape](https://github.com/unbescape/unbescape) which hasn't been updated for many years?

Reply:
Went with this as it's a smaller library and escaping HTML really shouldn't need to change often, if at all.